package/scripts/precook_test.py

```diff
@@ -0,0 +1,237 @@
+
+            # Define upper limit for random generation of integers for utime and stime
+            time_lim = 10000000
+            # 13 utime - Time in user mode
+            f.write(str(random.randint(1, time_lim)) + " ")
+            # 14 stime - Time in kernel mode
+            f.write(str(random.randint(1, time_lim)) + " ")
+            # 15 cutime - Time of waited-for children in user mode
+            f.write(str(random.randint(1, time_lim)) + " ")
+            # 16 cstime - Time of waited-for children in kernel mode
+            f.write(str(random.randint(1, time_lim)) + " ")
+
+            # 17 and 18 are never used -> set to 0
+            f.write("0 0 ")
+
+            # 19 num_threads - Number of threads of the process
+            num_threads = 1
+            f.write(str(num_threads) + " ")
+
+            # 20 never used -> set to 0
+            f.write("0 ")
+
+            # 21 starttime (uptime in utils.h) (last field that is read by any monitor)
+            f.write(str(random.randint(1, time_lim)))
+    else:
+        with open(stat_fname, "w") as f:
+            # Writing some pid and process name although never used
+            f.write(str(pid) + " ")  # 0 pid
+            f.write("(python3)" + " ")  # 1 Executable filename
+            # 2 to 12 are never used (as are 0 and 1) -> set to 0
+            for i in range(11):
+                f.write(str(0) + " ")
+
+            # 13 utime - Time in user mode
+            f.write(str(fixed_value) + " ")
+            # 14 stime - Time in kernel mode
+            f.write(str(fixed_value) + " ")
+            # 15 cutime - Time of waited-for children in user mode
+            f.write(str(fixed_value) + " ")
+            # 16 cstime - Time of waited-for children in kernel mode
+            f.write(str(fixed_value) + " ")
+
+            # 17 and 18 are never used -> set to 0
+            f.write("0 0 ")
+
+            # 19 num_threads - Number of threads of the process
+            num_threads = 1
+            f.write(str(num_threads) + " ")
+
+            # 20 never used -> set to 0
+            f.write("0 ")
+
+            # 21 starttime (uptime in utils.h) (last field that is read by any monitor)
+            f.write(str(fixed_value))
+
+
+def make_io(proc_pid, fixed_value, rand=False):
+    io_fname = os.path.join(proc_pid, "io")
+    if rand:
+        with open(io_fname, "w") as f:
+            char_lim = 1000
+            f.write("rchar: " + str(random.randint(0, char_lim)) + "\n")
+            f.write("wchar: " + str(random.randint(0, char_lim)) + "\n")
+            f.write("syscr: " + str(random.randint(0, char_lim)) + "\n")
+            f.write("syscw: " + str(random.randint(0, char_lim)) + "\n")
+            f.write("read_bytes: " + str(random.randint(0, char_lim)) + "\n")
+            f.write("write_bytes: " + str(random.randint(0, char_lim)) + "\n")
+            f.write("cancelled_write_bytes: " + str(random.randint(0, char_lim)) + "\n")
```

Comment:
Again, too much repetition, try:

```py
for field in "rchar:", "wchar:", ... 
    print(f"{field} {random.randint(0, char_lim)}, file=f)
```

You can also get rid of the whole `if rand` outer switch with something like:
```py
for field in "rchar:", "wchar:", ... 
    print(f"{field} {random.randint(0, char_lim) if rand else fixed_value}", file=f)
```